components/claim-mgt/org.wso2.carbon.identity.claim.metadata.mgt/src/main/java/org/wso2/carbon/identity/claim/metadata/mgt/ClaimMetadataManager.java

```diff
@@ -0,0 +1,214 @@
+package org.wso2.carbon.identity.claim.metadata.mgt;
```

Comment:
Check new classes and add the missing license.

Reply:
Fixed with 7c1f3db37ed9476b1d5e2ddaef02c8bf697bd03f